invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -407,6 +407,14 @@ <h4>{{ _("Request access") }}</h4>
 
 {%- block javascript %}
   {% if config.THEME_MATHJAX_CDN %}
+    <script type="text/javascript">
```

Comment:
move the entire block to `invenio-theme`, but pay attention to the `else` below `include config.THEME_JAVASCRIPT_TEMPLATE`